python/tests/integration/completions/test_text_completion.py

```diff
@@ -37,16 +37,38 @@
 
 from semantic_kernel import Kernel
 from semantic_kernel.connectors.ai.prompt_execution_settings import PromptExecutionSettings
+from semantic_kernel.exceptions import ServiceInitializationError
 from tests.integration.completions.completion_test_base import CompletionTestBase, ServiceType
 from tests.integration.completions.test_utils import retry
 
 ollama_setup: bool = False
 try:
-    if os.environ["OLLAMA_MODEL"]:
-        ollama_setup = True
-except KeyError:
+    OllamaTextCompletion()
+    ollama_setup = True
+except ServiceInitializationError:
     ollama_setup = False
 
+onnx_setup: bool = False
+try:
+    OnnxGenAITextCompletion()
```

Comment:
For my understanding: how come in the test chat completion base we do checks like `if os.environ[...]` but here we try to instantiate the object?

Reply:
good catch forgot to change it !